### ISSUE TITLE
Training code?

### ISSUE THREAD

**Issue by g1910** [TimoBolkart/voca#17]
Hi @TimoBolkart, any update on when the training code will be released?

Thanks

**Comment by kjhgfdsaas** [comment]
Hi @TimoBolkart 

I want to study how you applied the 'batch normalization' into the training stage.
Can you please release your training code? Maybe you can just release the 'Batch normalization' part.

Thanks

**Comment by TimoBolkart** [comment]
Training code is now added
